packages/labs/compiler/test_files/part_attribute.js

```diff
@@ -0,0 +1,2 @@
+import {html} from 'lit-html';
+const attributePart = html`<div data-attr="${'attributeValue'}"></div>`;
```

Comment:
Unsure how extensive this needs to be but should there also be examples of multiple expressions within the double quotes, and binding single value expression without double quotes?

Reply:
That's a great point. I can definitely add those!

Reply:
They don't need to be extremely exhaustive as eventually the lit-html_test suite will be compiled & tested which exercises many of these edge cases.

Reply:
Test updated